### DISCUSSION TITLE
Double click to open new tab for vertical tabs

### DISCUSSION THREAD

**Discussion by EmirYLMZ128** [pulse-browser/browser#260] [Ideas]
I've used plugins for vertical tabs for a long time and got used to opening new tabs by double clicking.  This is both faster and more convenient than clicking a button. Also the new tab button is far below.

on the settings page
Under Use vertical tabs (In dev)
use new tab button, use dobule click to open new tab
can be added

○ **Use vertical tabs**

○ use new tab button
○ use dobule click to open new tab
○ just use dobule click to open new tab (Hide new tab button)

**Comment by Ryosuke-Asano** [comment]
Did you try to use the shortcut key? (Ctrl + T)

**Reply by EmirYLMZ128** [reply]
I didn't use before, I always open a new tab by double clicking the tabbar.
but i think i can use shortcut now.

Ty
